airflow/migrations/versions/cf5dc11e79ad_drop_user_and_chart.py

```diff
@@ -47,7 +47,12 @@ def upgrade():  # noqa: D103
 
     if 'known_event' in tables:
         for fkey in inspector.get_foreign_keys(table_name="known_event", referred_table="users"):
-            op.drop_constraint(fkey['name'], 'known_event', type_="foreignkey")
+            if conn.dialect.name == "sqlite":
+                if fkey['name']:
+                    with op.batch_alter_table(table_name='known_event') as bop:
+                        bop.drop_constraint(fkey['name'], type_="foreignkey")
+            else:
+                op.drop_constraint(fkey['name'], 'known_event', type_="foreignkey")
```

Comment:
What if you use naming_convention? batch_alter_table supports naming_convention where unnamed foreign_keys are named during reflection. https://alembic.sqlalchemy.org/en/latest/batch.html
`naming_convention = {
    "fk":
    "fk_%(table_name)s_%(column_0_name)s_%(referred_table_name)s",
}`
```suggestion
            with op.batch_alter_table(table_name='known_event', naming_convention=naming_convention) as bop:
                bop.drop_constraint(fkey['name'], type_="foreignkey")

```

Reply:
Removed the conditional to check DB, went with the approach Tomek suggested